### PR TITLE
Socialite possibilities

### PR DESCRIPTION
**Pull request by rygos**
Adding Facebook and Twitter to the Possible Provider List.

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -13,6 +13,22 @@ CACHE_DRIVER=file
 SESSION_DRIVER=file
 QUEUE_DRIVER=database
 
+GITHUB_KEY=xxx
```
Comment (lianguan) [inline .env.example]: please don't use "xxx".
I don't recommend adding these in .env.example. we should introduce how to add them in the documentation.

**Comment by rygos** [comment]
i understand your critics for the "xxx" thing.
i would like it to see that we can add this information with Admin Panel.
Now you have to add this informations to the admin panel AND .env. This is not needed i tink.

File: .env.example (hunk 1 above)
Comment (lianguan) [inline .env.example]: This is also no needed.

**Comment by rygos** [comment]
I have deleted this .env.example Stuff.
I add this Informations to the Wiki.

**Pull request merged**